lib/modules/subredditInfo.js

```diff
@@ -204,6 +204,9 @@ async function showSubredditInfo(ele, context, update) {
 		const $subscribeToggle = $('#RESHoverInfoSubscriptionButton');
 		$subscribeToggle.attr('data-subreddit', jsonData.data.display_name.toLowerCase());
 		updateToggleButton($subscribeToggle, subscribed);
+		if (Modules.isEnabled(SubredditManager)) {
+			$subscribeToggle.after(SubredditManager.getMultiCounts(jsonData.data.display_name));
```

Comment:
the 'getMultiCounts' just returns a blank string if the count is 0, resulting in no extra HTML being added. If you have a better solution, I'm all for it